package.json

```diff
@@ -20,7 +21,8 @@
     "*.js": [
       "eslint --fix",
       "prettier --write"
-    ]
+    ],
+    "!(*.js)": "prettier --write --ignore-unknown"
```

Comment:
Prettier runs on all files, but separate `"*.js"` and `"!(*.js)"` tasks are needed to make sure that `eslint` runs before `prettier` on .js files.

https://github.com/lint-staged/lint-staged?tab=readme-ov-file#task-concurrency